tests/stub/driver_parameters/scripts/transaction_chaining.script

```diff
@@ -0,0 +1,61 @@
+!: BOLT 5.0
+!: ALLOW CONCURRENT
+!: ALLOW RESTART
+
+A: HELLO {"{}": "*"}
+*: RESET
+{*
+ {{
+    C: BEGIN {"tx_metadata": { "order": "1st" }, "[bookmarks]": "*", "[db]": "*" }
```

Comment:
Generally, it might be nice to give the metadata some human-readable value. Something like `{"order": "return bm1"}`, `{"order": "return no bookmark"}`, etc. This makes it easier to read the tests without switching between the Python file and the script constantly.

Reply:
The same thing also goes for `C: RUN "QUERY1" "*" "*"` in `tests/stub/driver_parameters/scripts/session_run_chaining.script`